packages/core/src/interfaces/charts.ts

```diff
@@ -627,4 +627,4 @@ export type ChartOptions =
 	| StackedBarChartOptions
 	| TreeChartOptions
 	| TreemapChartOptions
-	| WorldCloudChartOptions
+	| WordCloudChartOptions
```

Comment:
should we do `WordCloudChartOptions | WorldCloudChartOptions`? typescript could still throw an error not matching the type names

Reply:
Both are exported in packages/core/src/interfaces/index.ts which is the export point for the interfaces namespace. Does that address this concern?

Reply:
Updated.